streamz/sources.py

```diff
@@ -71,3 +72,42 @@ def start(self):
                     yield last
                 else:
                     return
+
+
+class Kafka(Source):
+    """ Accepts messages from Kafka
+
+    Parameters
+    ----------
+    topics: list of str
+        Labels of Kafka topics to consume from
+    url: str
+        Connection string (host:port) by which to reach Kafka
+    group: str
+        Identity of the consumer. If multiple sources share the same group,
+        each message will be passed to only one of them.
+    poll_interval: number
+        Seconds that the thread sleeps between polling Kafka for new messages
```

Comment:
Slight nit, the thread doesn't sleep.  I would just say "seconds between polling kafka for new messages"